rusoto/core/src/param.rs

```diff
@@ -5,11 +5,14 @@
 //! Supports optional parameters for calling SQS and ETS.
 
 use std::collections::BTreeMap;
+/// Paramaters for HTTP Request stored as a `BTreeMap`
 pub type Params = BTreeMap<String, Option<String>>;
 
 /// Key:value pair for an service parameter.
 pub trait ServiceParams {
+    /// Add a new paramater with a key and val
```

Comment:
I'm always a fan of leaving documentation on each param (even if it seems obvious):

```rust
/// Adds a new Parameter.
///
/// * `key` - The key of the parameter to add.
/// * `val` - The value of the parameter to add.
```